packages/tenant-process/src/services/readModelService.ts

```diff
@@ -565,6 +567,87 @@ export function readModelServiceBuilder(
         "data.kind": delegationKind.delegatedConsumer,
       });
     },
+    async getTenantVerifiedAttributeVerifiers(
+      tenantId: TenantId,
+      attributeId: AttributeId,
+      { offset, limit }: { offset: number; limit: number }
+    ): Promise<ListResult<TenantVerifier>> {
+      const tenant = await tenants.findOne({ "data.id": tenantId });
+      if (!tenant) {
+        return { results: [], totalCount: 0 };
+      }
+
+      const verifiedAttribute = tenant.data.attributes
+        .filter((attr) => attr.type === "PersistentVerifiedAttribute")
+        .find((attr) => attr.id === attributeId);
+
+      if (
+        !verifiedAttribute ||
+        verifiedAttribute.type !== "PersistentVerifiedAttribute"
```

Comment:
is check on `type` required since we already have [this](https://github.com/pagopa/interop-be-monorepo/pull/2175/files#diff-c572b9bb95c4fcf3227c7b8343252d56aedb4e17e3f5d299bbfe69068a41f5f6R581) filter?